docs/cli_demo/bitcoind.md

```diff
@@ -0,0 +1,284 @@
+# Working with `Bitcoind`
+
+In this tutorial, we will guide you through setting up a Bitcoin Core `bitcoind` node in `regtest` mode. We'll cover basic operations like creating wallets, generating blocks, checking balances, and sending Bitcoin between two wallets.
```

Comment:
Add a line saying `regtest` is a toy network, where we can create our own blocks and generate coins. which is useful for easy integration testing of the apps.

The actual swap market exists in Signet.  Network can be changed by changing the `bitcoin.conf` as explained below.